src/nupic/regions/PyRegion.py

```diff
@@ -273,6 +273,25 @@ def setParameterArray(self, name, index, array):
     array: the value to assign to the requested parameter (a numpy array)
     """
 
+
+  def write(self, proto):
+    """This method is called during network serialization with a proto object
+    so that this region can write its data to it.
+
+    proto: PyRegionProto capnproto object
+    """
+    pass
```

Comment:
I don't think so, because there can be subclasses of this that don't need serialization.